apps/dashboard/app/(app)/settings/billing/plans/page.tsx

```diff
@@ -12,35 +12,35 @@ const tiers = {
     name: "Free Tier",
     href: "/app",
     price: 0,
-    description: "Everything you need to start your next API!",
+    description: "Everything you need to start!",
     buttonText: "Free",
     features: [
-      "100 Monthly Active Keys",
-      "2500 Successful Verifications per month",
-      "100k Successful Ratelimits per month",
+      "1k API keys",
+      "2.5k successful verifications / month",
+      "100k successful ratelimits / month",
+      "7-day analytics retention",
       "Unlimited APIs",
-      "7 Days Analytics Retention",
     ],
     footnotes: [],
   },
   pro: {
     name: "Pro Tier",
     href: "/app",
     price: 25,
-    description: "For those with teams and more demanding needs",
+    description: "For growing teams with powerful demands",
     buttonText: "Pro",
     features: [
-      "250 Monthly Active keys included *",
-      "150,000 Successful Verifications included **",
-      "2,500,000 Successful Ratelimits included ***",
+      "1M API keys *",
+      "150k successful verifications / month **",
+      "2.5M successful ratelimits / month ***",
+      "90-day analytics retention",
+      "90-day audit log retention",
       "Unlimited APIs",
       "Workspaces with team members",
-      "90 Days Analytics Retention",
-      "90 Days Audit Log Retention",
     ],
     footnotes: [
       "*  Additional active keys are billed at $0.10",
```

Comment:
1mil+ keys will be a custom enterprise contact, there's no need for an asterisk here
```suggestion
   
```